src/blocks/form/test/form.cypress.js

```diff
@@ -6,6 +6,8 @@
  */
 import * as helpers from '../../../../.dev/tests/cypress/helpers';
 
+// TODO : there seems to be a real bug here, as the success message is not shown and the form is still there
+
```

Comment:
Let's follow up on this. I could not replicate this.

Reply:
Oh! Will try that first thing tomorrow morning when I'll be back!

Reply:
The form tests are passing for you? Here, in local, with beta3, I still have this error. If it passes on local for you and on the CI, it's probably a weirdness of my computer ;)

Reply:
Locally for these form tests to pass it would require additonal configuration of the MailHog software. This is handled within the e2e enviornment by bash script so generally you would see differing results from From block tests locally and remote.